README.md

```diff
@@ -1,2 +1,148 @@
-# My-Portfolio
-My personal portfolio.
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#Thi is my personal portfolio)
+  - [🛠 Built With](#HTML-CSS)
+    - [Tech Stack](#HTNL-CSS)
+    - [Key Features](#About-me)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [My-Portofolio] <a name="This is my personal portfolio"></a>
+
+**[My-Portofolio]** is a program for my coding projects.
+
+## 🛠 Built With <a name="built-with"></a>
+
+ - HTML
+ - CSS
+
+
+### Key Features <a name="About-me"></a>
+
+- **[Use_HTML]**
+- **[Use_CSS]**
+- **[Use_Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need: 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Usage
+
+To run the project, execute the following steps: 
+```sh
+  Open the local folder where your cloned your project to.
+  Locate the html file named index.html and right click on it.
+  Select open with and choose google chrome.
+```
+
+
+### Deployment
+
+You can deploy this project using: google chrome.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+- All collaborators
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Leeoasis)
+- Twitter: [@twitterhandle](https://twitter.com/gudo_leslie)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/leslie-gudo-b08a49255/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Projects]**
+- [ ] **[About_my_self]**
+- [ ] **[Contact_me]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Leeoasis/My-Portfolio/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
```

Comment:
- [ OPTIONAL] Some of the titles in your readme file like `show your support` and `acknowledgements` are empty. Kindly add some content to them. The `FAQ` section is also optional. You can remove it if for now.